internal/k8s/handlers.go

```diff
@@ -468,6 +468,48 @@ func createPolicyHandlers(lbc *LoadBalancerController) cache.ResourceEventHandle
 	}
 }
 
+func createNginxCisConnectorHandlers(lbc *LoadBalancerController) cache.ResourceEventHandlerFuncs {
+	return cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			con := obj.(*unstructured.Unstructured)
```

Comment:
Do you need to handle the cast failing like below in the DeleteFunc case?

Reply:
this is only necessary for the Delete function. From the doc: `DeletedFinalStateUnknown is placed into a DeltaFIFO in the case where an object was deleted but the watch deletion event was missed while disconnected from apiserver`  https://github.com/kubernetes/client-go/blob/master/tools/cache/delta_fifo.go#L736